codegen/src/main/scala/com/soundcloud/twinagle/codegen/ServerClientCodeGenerator.scala

```diff
@@ -43,7 +43,8 @@ object ServerClientCodeGenerator extends CodeGenApp {
           for {
             file        <- request.filesToGenerate
             serviceFile <- generateServiceFiles(file, implicits)
-          } yield serviceFile
+          } yield serviceFile,
+          supportedFeatures = Set(CodeGeneratorResponse.Feature.FEATURE_PROTO3_OPTIONAL)
```

Comment:
is this empty by default? If not, maybe
```suggestion
          supportedFeatures += CodeGeneratorResponse.Feature.FEATURE_PROTO3_OPTIONAL
```

Reply:
[It's empty](https://github.com/scalapb/protoc-bridge/blob/master/protoc-gen/src/main/scala/protocgen/CodeGenResponse.scala#L23), so I think it's probably fine to leave as-is?